### ISSUE TITLE
Good To have: Initial Position

### ISSUE THREAD

**Issue by tombby11** [TorbenK/TK.CustomMap#225]
It is not an issue rather a "Good to have" 
If you build the map in the xaml code , the map will start with the default position (which is in my iOS case Rome) . The problem is that if you have the map in xaml then you cannot access the constructor with MapSpan parameter. 
So it would be nice to have a property called InitialMapRegion which is read only once when the map is instantiated. 

This problem obviosly can be solved by implementing the map in the code behind,  but then you lose the whole point of have xaml.

**Comment by RocketPop** [comment]
This seems to be worse than that. Binding the MapCenter from xaml doesn't appear to work at all for the first display of the map. If I navigate back to an already created map - it seems to work better. To repro - just create a map in xaml, with a binding to Postion property - all I see is Rome

**Comment by adrianclaveri** [comment]
I believe the issue is due to updating the UI in the constructor of the view model that is binded to the view. If you set the MapRegion in a separate thread it will do the trick. An example:
Device.BeginInvokeOnMainThread(() =>
			{
                MapRegion = MapSpan.FromCenterAndRadius(new Position(-34.7969179, -56.0670389), Distance.FromKilometers(1));
			});

**Comment by RocketPop** [comment]
The issue here is more about how {Binding ...} to various properties doesn't work very well - but I might try that in my code-behind hack :)

**Comment by vincentcastagna** [comment]
It seems its not a TK issue. Comes from XF Maps : https://forums.xamarin.com/discussion/25537/android-googlemap-resets-camera-to-rome

**Comment by gpatkos** [comment]
Initializing the map start position from xaml is working fine for me. 

```
<customMaps:TKCustomMap
        x:Name="MainMap"
        IsVisible="{Binding IsSearchModeEnabled, Converter={StaticResource NegationConverter}"
        AbsoluteLayout.LayoutFlags="All" 
        AbsoluteLayout.LayoutBounds="0,0,1,1"
        MapType="Street"
        IsShowingUser="False"
        MapCenter="{Binding MapCenter}"
        CustomPins="{Binding CustomPins}"
        MapRegion="{Binding MapRegion}"
        PinSelectedCommand="{Binding PinSelectedCommand}"
        SelectedPin="{Binding SelectedPin}"
        HorizontalOptions="FillAndExpand"
        VerticalOptions="FillAndExpand"
        IsRegionChangeAnimated="True">
        
        <x:Arguments>
          <maps:MapSpan x:FactoryMethod="FromCenterAndRadius">
            <x:Arguments>
              <maps:Position>
                <x:Arguments>
                  <!-- Budapest 0 km point -->
                  <x:Double>47.513577</x:Double>
                  <x:Double>19.0500007</x:Double>
                </x:Arguments>
              </maps:Position>
              <maps:Distance x:FactoryMethod="FromMeters">
                <x:Arguments>
                  <x:Double>10000</x:Double>
                </x:Arguments>
              </maps:Distance>
            </x:Arguments>
          </maps:MapSpan>
        </x:Arguments>

</customMaps:TKCustomMap>
```
Namespace references:
```
xmlns:maps="clr-namespace:Xamarin.Forms.Maps;assembly=Xamarin.Forms.Maps"
xmlns:customMaps="clr-namespace:TK.CustomMap;assembly=TK.CustomMap"
```
